admin_guide/securing_builds.adoc

```diff
@@ -27,6 +27,13 @@ xref:../architecture/core_concepts/builds_and_image_streams.adoc#docker-build[Do
 build] permission should also be granted with caution as a vulnerability in the Docker build
 logic could result in a privileges being granted on the host node.
 
+[WARNING]
+====
+During builds, actions performed by the Docker daemon are privileged and occur in the host
+network namespace. Such actions bypass configured networking rules including
```

Comment:
The warning is completely fine but perhaps we should be more explicit saying this only affects docker builds and potentially custom builds to avoid customers being concerned about non real problems.
Could  we replace "During builds" with "During docker and custom builds"?

Reply:
Thanks, fixed.